public/expenses/edit.php

```diff
@@ -19,6 +19,9 @@
 $title = 'Edit Expense';
 include public_path('layouts/header.php');
 ?>
+<head>
```

Comment:
`<head>` tag element cannot be re-opened. Just include the Js script at the end of the page and the browser would load its and compile its content.

Reply:
tag element can be re-opened.